cocos/platform/android/java/src/org/cocos2dx/lib/Cocos2dxGLSurfaceView.java

```diff
@@ -198,8 +197,7 @@ public void run() {
                 Cocos2dxGLSurfaceView.this.mCocos2dxRenderer.handleOnPause();
             }
         });
-        this.setRenderMode(RENDERMODE_WHEN_DIRTY);
-        // super.onPause();
+        super.onPause();
```

Comment:
this change should be tested well. Has had issues in the past. Will this whole change work if commented out ?

Reply:
https://github.com/cocos2d/cocos2d-x/issues/19992

Reply:
can you test the text rendering issues? then we can go live